sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -189,6 +189,7 @@ message ProcessBundleRequest {
 
 // Stable
 message ProcessBundleResponse {
+  // (Required)
```

Comment:
I wouldn't say that this is required, I think a comment stating that if metrics is supported by the SDK harness, then this response here represents the 'final' metrics to record for this bundle.

Reply:
I was going for empty metrics response but this makes more sense.